src/ImageSharp/Formats/Gif/GifEncoder.cs

```diff
@@ -30,6 +30,14 @@ public sealed class GifEncoder : IImageEncoder, IGifEncoderOptions
         /// </summary>
         public IQuantizer Quantizer { get; set; } = new OctreeQuantizer();
 
+        /// <summary>
+        /// Gets or sets a value indicating whether pixels that stay the same between two frames should be reused
+        /// by forcing the <see cref="DisposalMethod"/> of all frames to <see cref="DisposalMethod.NotDispose"/>
+        /// and making the repeated pixel transparent. This improves GIF quality when pixels are
+        /// repeated across frames.
+        /// </summary>
+        public bool CutRepeatedPixels { get; set; } = false;
```

Comment:
I defaulted this to `false` because it overrides the `DisposalMethod` of frames which might be unexpected. Not sure about this behavior though.

Reply:
I don't think this should be a member of the options and should only be determined by the `DisposalMethod` enumeration.